README.md

```diff
@@ -44,3 +44,60 @@ module.exports = {
 ```bash
 yarn jest
 ```
+
+
+## Options
+
+This project uses [cosmiconfig](https://github.com/davidtheclark/cosmiconfig), so you can provide config via:
+* a `jest-runner-eslint` property in your `package.json`
+* a `jest-runner-eslint.config.js` JS file
+* a `.jest-runner-eslintrc` JSON file
+
+
+In `package.json`
+```json
+{
+  "jest-runner-eslint": {
+    "cliOptions": {}
+  }
+}
+```
+
+See https://eslint.org/docs/developer-guide/nodejs-api#cliengine for a full
+set of supported `cliOptions`
+
+#### `fix`
+This is similar to running ESLint with `--fix`
+```json
+{
+  "jest-runner-eslint": {
+    "cliOptions": {
+      "fix": true
+    }
+  }
+}
+```
+
+#### `configFile`
+This is similar to running ESLint with `-c /path/to/config-file`
+```json
+{
+  "jest-runner-eslint": {
+    "cliOptions": {
+      "configFile": "/path/to/config-file"
```

Comment:
similarly, this should be `config`, since it should map to actual CLI options, and not to eslint's API (that very few users ever interact with)